plugins/modules/grafana_datasource.py

```diff
@@ -65,6 +67,12 @@
   password:
     description:
     - The datasource password.
+    - For encrypted password use C(secure_password).
+    type: str
+  secure_password:
```

Comment:
:( but you're the maintainer. I thought it would be a more user-friendly way.
I will change it after we settled in the enforce_secure_data conversation.

Reply:
Can we at least agree on a note in the option description for password? #113 proves that there is not only me getting confused.